uber/site_sections/hotel.py

```diff
@@ -39,6 +39,11 @@ def hours(self, session):
         staffers = [s for s in staffers if s.hotel_shifts_required and s.weighted_hours < 30]
         return {'staffers': staffers}
 
+    def no_shows(self, session):
+        staffers = session.query(Attendee).filter_by(badge_type=STAFF_BADGE).order_by(Attendee.full_name).all()
+        staffers = [s for s in staffers if s.hotel_shifts_required and not s.checked_in]
```

Comment:
Oh, good point.  We should instead say ``.hotel_nights`` which will return a falsey value if they don't have any nights assigned, and a truthy value if there are hotel requests.